replay: iframe srcdoc block (#7046)
getsentry/sentry-docs: src/platforms/javascript/common/session-replay/troubleshooting.mdx

More details about this feature can be found in the <PlatformLink to="/session-replay/configuration/">configuration page</PlatformLink>.

> ## Text in the iframe is not getting masked

Reviewer: ```suggestion
## Text in the iframe isn't masked
```

> If your iframe relies on `srcdoc`, the contents are taken as-is. As opposed to loading a resource via `src`. For that reason, our masking logic doesn't run through it.

Reviewer: ```suggestion
Our masking logic doesn't run on iframe content that's provided using the `srcdoc` attribute, rather than loaded in via `src`.

To hide this content, block the iframe, as described in our Session Replay [Privacy](platforms/session-replay/privacy/#blocking) docs.
```

> <PlatformLink to="/session-replay/privacy/">
>   In such case it's best to block the iframe
> </PlatformLink>
> .

Reviewer: ```suggestion
```
